### ISSUE TITLE
Cannot manually turn the vibrate and sound on and off

### ISSUE THREAD

**Issue by framirez123** [zo0r/react-native-push-notification#1373]
I am trying to manually turn the vibrate and sounds off by doing this code
`PushNotification.localNotification({
                importance: "low",
                message: "Triage Received",
                playSound: false, 
                soundName: 'test',
                vibrate: true,
                Vibration: 0
                });`
But phone still make a sound and vibrate. Is this a bug?
same with this issue #1362

**Comment by Dallas62** [comment]
Hi @framirez123 
Same answer:

Thanks for your report,
I will not be able to test this on a real device so if you can test this PR #1042 it would be great!

Change the current version of `react-native-push-notification`:
```
    "react-native-push-notification": "git+https://github.com/fleeteng/react-native-push-notification.git"
````
then run:
```
yarn install
```
**OR**
```
npm install
```
once it's done, you should be able to test the PR, if it's good for you, I will merge the PR.

**Comment by helloeyesimple** [comment]
@Dallas62 PR #1042 not working, phone still making sound.
Samsung Galaxy S7 Edge Android 8.0.0

**Comment by framirez123** [comment]
@Dallas62 
Sorry for late reply. Was working on something else.
The one you recommended didn't seem to work. 
At first is it wont run. Then when if finally run. It still vibrates and not silent.
Used devices : 
Huawei Y7
Real me 3
Samsung S7

**Comment by Dallas62** [comment]
Hi @framirez123 
I will receive a development phone soon, and test this issue ASAP.

**Comment by Dallas62** [comment]
Seems to be fix in `3.3.1`.

**Comment by framirez123** [comment]
Okay. Thanks for the support. Will check later then. Cheers
